addons/account/wizard/account_move_send_views.xml

```diff
@@ -15,16 +15,20 @@
                 <field name="enable_download" invisible="1"/>
                 <field name="enable_send_mail" invisible="1"/>
                 <field name="send_mail_readonly" invisible="1"/>
+                <field name="send_mail_warning_message" invisible="1"/>
                 <field name="display_mail_composer" invisible="1"/>
                 <field name="mail_lang" invisible="1"/>
-                <field name="mail_partner_ids" widget="many2many_tags_email" class="d-none" context="{'force_email': True}"/>
 
                 <!-- Warnings -->
                 <div name="warnings">
-                    <div class="alert alert-warning"
-                        role="alert"
-                        invisible="not send_mail_warning_message">
-                        <field name="send_mail_warning_message"/>
+                    <span invisible="send_mail_warning_message != 'missing_info'"><b>E-mail</b></span>
+                    <div class="alert alert-warning" role="alert"
+                        invisible="send_mail_warning_message != 'missing_info'">
+                        Cannot send an email to some of the
+                        <button name="action_open_partners_without_email"
+                            type="object"
+                            class="btn btn-link p-0 align-baseline fst-italic">partners</button>
+                        because they don't have an email address.
```

Comment:
```suggestion
                        Cannot send an email to 
                        <button name="action_open_partners_without_email"
                            type="object"
                            class="btn btn-link p-0 align-baseline fst-italic">some of the partners</button>
                        because they don't have an email address.
```

Reply:
@lordkrandel We could not have send_mail_warning_message as an html field instead?  The "missing_mail" code is a bit hacky.  (it could be a simple computed boolean as well)

Reply:
Odoo 17 is to be considered stable now. I cannot change field definitions, right?